src/ralph/dhcp/views.py

```diff
@@ -57,6 +58,16 @@ def dispatch(self, request, *args, **kwargs):
 class DHCPConfigMixin(object):
     content_type = 'text/plain'
 
+    @staticmethod
+    def check_existence(model_class, names):
+        found = [
```

Comment:
not exactly cause I using objects in https://github.com/allegro/ralph/pull/2399/files#diff-a97db8bac29d9b8096414a7656131436R89 and https://github.com/allegro/ralph/pull/2399/files#diff-a97db8bac29d9b8096414a7656131436R100 but simple filter is better than strange list comprehension